src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/BackupController.php

```diff
@@ -137,7 +137,9 @@ public function downloadContentAction($downloadFileName)
      * Process backup options saving.
      *
      * @AdminSecurity(
-     *     "is_granted('update', request.get('_legacy_controller')) && is_granted('create', request.get('_legacy_controller')) && is_granted('delete', request.get('_legacy_controller'))"
+     *     "is_granted('update', request.get('_legacy_controller')) && is_granted('create', request.get('_legacy_controller')) && is_granted('delete', request.get('_legacy_controller'))",
```

Comment:
it seems you can use `and` instead of `&&`
But the result is the same

Reply:
And I searched for https://github.com/symfony/security/blob/4.4/Core/Authorization/AuthorizationChecker.php#L59 to be sure :D

Reply:
So as we spoke, to be consistent with the whole code I will keep the '&&', anyway thanks for the advise.